testing/test_unittest.py

```diff
@@ -718,3 +718,16 @@ def test_one(self):
         *SKIP*[1]*test_foo.py*skipping due to reasons*
         *1 skipped*
     """)
+
+def test_unittest_skip_issue1169(testdir):
+    testpath = testdir.makepyfile(test_foo="""
+        import unittest
+        
+        class MyTestCase(unittest.TestCase):
+            @unittest.skip
+            def test_skip(self):
+                 self.fail()
+        """)
+    reprec = testdir.inline_run(testpath)
+    reprec.assertoutcome(passed=1)
```

Comment:
>  I think it is the outcome of the test case rather than the test case's methods.

Hmm that's strange then, `assertoutcome` gives the outcome of the test session, so if this is working as it should then you should see `skipped=1`.

I just tried your branch and noticed you should use `@unittest.skip("some reason")` instead of a plain `@unittest.skip` decorator, that's why it was passing instead of just skipping, the `reason` parameter is not optional. As I have already applied the changes myself, let me commit my change and your PR will be merged once I push to `master`.

Thanks again!